docs/Themes.md

```diff
@@ -0,0 +1,333 @@
+# Themes
+
+Themes are used to customise the colors of the card. github-readme-testaustime ships with a whopping 60 themes to choose from.
```

Comment:
We could just link the upstream's theme listing <https://github.com/anuraghazra/github-readme-stats/blob/678fa1141676fc8fb75e2a322e4297323900e330/themes/README.md>